src/hierarchical-menu/hierarchical-menu.ts

```diff
@@ -69,13 +69,13 @@ export class NgAisHierarchicalMenu extends BaseWidget {
 
   constructor(
     @Inject(forwardRef(() => NgAisInstantSearch))
-    public instantSearchParent: any
+    public instantSearchParent: NgAisInstantSearch
   ) {
     super('HierarchicalMenu');
   }
 
   public ngOnInit() {
-    this.createWidget(connectHierarchicalMenu, {
+    this.createWidget(connectHierarchicalMenu as Connector, {
```

Comment:
this probably can be fixed in InstantSearch.js, but I didn't look yet how, I think it was because a type was `Function` instead of a more detailed callback like the other ones.

Reply:
How come there was no error before?

Reply:
before hierarchical menu was a .js file, now it has the .d.ts